src/modules/licence-import/mappers/document.js

```diff
@@ -0,0 +1,53 @@
+const { groupBy, sortBy, last, identity } = require('lodash');
+const date = require('./date');
+const str = require('./str');
+
+const mapExternalId = (row) =>
+  `${row.FGAC_REGION_CODE}:${row.AABL_ID}:${row.ISSUE_NO}:${row.INCR_NO}`;
+
+const statuses = {
+  CURR: 'current',
+  SUPER: 'superseded',
+  DRAFT: 'draft'
+};
+const mapStatus = status => statuses[status];
+
+const getDocumentEndDate = (licenceVersions, licence) => {
+  // Map to a document
+  // the start date is from the first increment, and the end date from the last
+  const endDates = [
+    licence.endDate,
+    last(licenceVersions).EFF_END_DATE
+  ]
+    .map(str.mapNull)
+    .filter(identity)
+    .map(date.mapNaldDate);
+
+  return date.getMinDate(endDates);
+};
+
+const mapDocuments = (data, licence) => {
+  // Remove draft rows
+  const filtered = data.filter(row => row.STATUS !== 'DRAFT');
```

Comment:
Do you have to filter DRAFT out here even though your SQL WHERE clause  includes <> 'DRAFT' on line 35 in file src/modules/licence-import/queries/index.js

Reply:
I have made the `getLicenceVersions` query ignore drafts and removed this now